includes/class-amp-story-post-type.php

```diff
@@ -219,4 +214,21 @@ public static function filter_template_include( $template ) {
 		}
 		return $template;
 	}
+
+	/**
+	 * Add CSS to the frontend for AMP Stories.
+	 *
+	 * @see /assets/css/amp-story-frontend.css
+	 */
+	public function add_frontend_css() {
```

Comment:
This should probably be `static` if it's statically called.

Reply:
👍